src/components/GameListCard/GameListCard.tsx

```diff
@@ -0,0 +1,49 @@
+'use client'
+import Image from 'next/image'
+
+interface gameCardProps {
+  title: string
+  description?: string
+  src?: string
+  onClick?: () => void
+}
+
+const GameListCard = ({
+  title,
+  description,
+  src,
+  onClick,
+  ...props
+}: gameCardProps) => {
+  return (
+    <article
+      className={`basis-1/3 max-w-80 min-w-60 relative bg-primary-container shadow rounded-md overflow-hidden`}
+      onClick={onClick}
+      {...props}
+    >
+      {src && (
+        <Image
+          src={src}
+          alt={title}
+          layout="fill"
+          objectFit="cover"
+          className="absolute"
+        />
```

Comment:
P3; layout, objectFit의 경우 13버전에서 deprecated 되었습니다!
관련 [Nextjs 한글 번역 문서](https://nextjs-ko.org/docs/app/api-reference/components/image#%EB%B2%84%EC%A0%84-%ED%9E%88%EC%8A%A4%ED%86%A0%EB%A6%AC) 
```suggestion
        <Image
          src={src}
          alt={title}
          fill
          style={{ objectFit: "cover" }}
          className="absolute"
        />
```
위와 같이 바뀌야 하는 것으로 알고 있습니다!

Reply:
첨부된 링크 확인해보겠습니다~